src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java

```diff
@@ -33,7 +33,10 @@ public void EditPreferences_001_selectVE() {
     EditPreferencesPage editPrefPage = new EditPreferencesPage(driver).openEditingSection();
     editPrefPage.selectPreferredEditor(VE);
     PreferencesPageObject prefPage = editPrefPage.clickSaveButton();
-    prefPage.getBannerNotifications().verifyNotificationMessage();
+
+    Assertion.assertTrue(prefPage.getBannerNotifications().isNotificationMessageVisible(),
+                         "Notification message is not visible");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Notification message is not visible" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)